native/src/components/OpeningHours.tsx

```diff
@@ -61,6 +61,7 @@ type OpeningHoursProps = {
   openingHours: OpeningHoursModel[] | null
   isTemporarilyClosed: boolean
   appointmentUrl: string | null
+  link: string | null
```

Comment:
Because there are several cases here: If a location has a dedicated appointment url, we use that for the `link`. If it doesn't have a dedicated appointment url but does have a website url, we use that. And if it doesn't have either of those, we don't show a link.

That means that we shouldn't use the `appointmentUrl` because in the case that we don't have a dedicated url to make an appointment, but do have one to the general website, we would be leaving out information for the user.

Reply:
Hmmm... I don't get that. Why does it make sense to show the website url in the tooltip but not as a link if no specific appointment url is set? What is the difference here? From a user perspective, I think this just makes it more complicated by not directly showing the link. Instead, I have to hover the icon, move my mouse correctly and then click the link instead of just clicking a clear link.
In both cases, the user has to set an appointment via the website anyway,

If this really makes sense, please still rename `link` to something more speaking, e.g. `website`.